simapp/example_app.yaml

```diff
@@ -0,0 +1,47 @@
+modules:
+  - name: runtime
+    config:
+      "@type": cosmos.app.runtime.v1alpha1.Module
+      app_name: SimApp
+      # During begin block slashing happens after distr.BeginBlocker so that
+      # there is nothing left over in the validator fee pool, so as to keep the CanWithdrawInvariant invariant.
+      # NOTE: staking module is required if HistoricalEntries param > 0
+      begin_blockers: [distribution, staking]
```

Comment:
do we need preblockers?

Reply:
yes

Reply:
i'll rewrite it to match simapp entirely